service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/AzureDatabaseUtilsRunner.java

```diff
@@ -381,6 +423,35 @@ private void runAzureDatabaseUtils(
           workspaceId,
           namespace);
       deleteContainer(aksApi, safePodName, namespace);
+      deleteSecret(aksApi, safePodName, namespace);
```

Comment:
Will this error if no secret has been created?

Reply:
it shouldn't error, just log a warning. I think @dvoet 's previous comment suggested doing it this way. I don't have a strong opinion about it (or I possibly misinterpreted that comment)